ansys/dpf/core/plotter.py

```diff
@@ -391,6 +391,8 @@ def add_mesh(self, meshed_region, deform_by=None, scale_factor=1.0, **kwargs):
         >>> pl.add_mesh(mesh)
 
         """
+        if meshed_region.grid is not None:
```

Comment:
Have you checked what happens with meshes containers?

Reply:
It works.